keys.go

```diff
@@ -362,6 +362,24 @@ func (c *Client) Rotate(ctx context.Context, id, payload string) error {
 	return nil
 }
 
+// Disable a key
```

Comment:
Probably want to link to documentation to explain Enable/Disable of keys.

https://cloud.ibm.com/docs/key-protect?topic=key-protect-disable-keys